<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculateWidget</class>
 <widget class="QWidget" name="CalculateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculateWidget</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#returnButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#returnButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>255</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QPushButton" name="refleshButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#refleshButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#refleshButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#searchButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#calculateButton:!hover
{
border-radius:10px;
background-color:rgb(255,255,143);
}
#calculateButton:hover
{
border-radius:10px;
background-color:rgb(255,100,143);
}</string>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="jobShowLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="totalSalShowLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="typeInLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="numberLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="totalSalLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>当月工资：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="typeInShowLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="numberLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);
font: 9pt &quot;新宋体&quot;;</string>
      </property>
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
